gio/src/subclass/file_monitor.rs

```diff
@@ -0,0 +1,85 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{subclass::prelude::*, File, FileMonitor, FileMonitorEvent};
+use glib::{prelude::*, translate::*};
+
+pub trait FileMonitorImpl: ObjectImpl {
```

Comment:
Is this actually useful?

Reply:
Today it is not for sure. My intention is to cover up whatever APIs we can manually to have a better understanding of what is needed for an automatically generated of the subclassing code in the future

Reply:
I mean, is this actually intended to be subclasses? The API exposed here looks quite incomplete for implementing a new file monitor